reactivesearch/README.md

```diff
@@ -6,13 +6,15 @@ Step-by-step guide available at [ReactiveSearch Quickstart Doc](https://docs.app
 
 ### Configure
 
-The ReactiveSearch components code resides in `src/App.js` file.   
+The ReactiveSearch components code resides in `src/App.js` file.  
+**Note that this requires Node.js < v17** 
```

Comment:
That's interesting!   Surprised that they haven't upgraded mroe!

Reply:
Node.js stopped supporting `NODE_OPTIONS=--openssl-legacy-provider` in v18.  I'm not sure why that's used to run it all, but I didn't want to mess with it